cmd/collector/app/builder_flags.go

```diff
@@ -88,8 +100,11 @@ func (cOpts *CollectorOptions) InitFromViper(v *viper.Viper) *CollectorOptions {
 	cOpts.NumWorkers = v.GetInt(collectorNumWorkers)
 	cOpts.CollectorHTTPPort = v.GetInt(collectorHTTPPort)
 	cOpts.CollectorGRPCPort = v.GetInt(collectorGRPCPort)
+	cOpts.CollectorHTTPHostPort = v.GetString(collectorHTTPHostPort)
+	cOpts.CollectorGRPCHostPort = v.GetString(collectorGRPCHostPort)
+	cOpts.CollectorZipkinHTTPPort = v.GetInt(collectorZipkinHTTPPort)
+	cOpts.CollectorZipkinHTTPHostPort = v.GetString(collectorZipkinHTTPHostPort)
```

Comment:
I prefer we take care of deprecated flags right here, not propagate them down to the service APIs.